vaadin-context-menu-flow-integration-tests/webpack.generated.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Looks like you committed this file by accident.

Reply:
Thanks, removed.